pkg/aggregator/internal/limiter/limiter.go

```diff
@@ -47,6 +56,20 @@ func New(limit int, keyTagName string, telemetryTagNames []string) *Limiter {
 		return nil
 	}
 
+	return newLimiter(limit, math.MaxInt, 0, keyTagName, telemetryTagNames)
+}
+
+// NewGlobal returns a limiter with a global per-instance limit, that
```

Comment:
If we're calling this "global", then maybe the regular `New` should be `NewPerSender` for clarity.

Either way, the `global per-instance limit` verbiage is confusing to me. 
```suggestion
// NewGlobal returns a limiter with a global limit which will be equally split between senders
```